### PR TITLE
add prefer-lowest to travis

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,10 +25,14 @@ matrix:
...
       env: CODECOVERAGE=1 DEFAULT=0
 
+    - php: 5.6
+      env: PREFER_LOWEST=1
+
   fast_finish: true
 
 before_script:
-  - composer install --prefer-source --no-interaction
+  - sh -c "if [ '$PREFER_LOWEST' != '1' ]; then composer install --prefer-source --no-interaction; fi"
```
Comment (dereuromark) [inline .travis.yml]: the sc -c part shouldnt be necesarray anymore

**Comment by dereuromark** [comment]
Any idea why the recent CakePHP versions have a serious runtime performance issue with older (PHP5.6) PHP versions? "1000 iterations should be as fast as around 0.1-0.2 sek, but are 0.63sek."
Only there, in newer PHP versions all good. Maybe we have to relax the test then...

File: .travis.yml (hunk 1 above)
Comment (saeideng) [reply .travis.yml]: I copied from debug_kit travis.yml :)

**Comment by saeideng** [comment]
on php 7.2 https://travis-ci.org/dereuromark/cakephp-tinyauth/jobs/341053326#L523

**Comment by dereuromark** [comment]
Super slow sometimes, but maybe travis fault.

**Pull request merged**